config/config.go

```diff
@@ -91,7 +91,7 @@ type Config struct {
 	NydusdBinaryPath         string        `toml:"nydusd_binary_path"`
 	NydusImageBinaryPath     string        `toml:"nydus_image_binary"`
 	DaemonMode               DaemonMode    `toml:"daemon_mode"`
-	FsDriver                 string        `toml:"daemon_backend"`
+	FsDriver                 string        `toml:"fs_driver"`
```

Comment:
Does this break compatibility?

Reply:
No. We don't have configuration file used in fact.